src/pool-priority-sort.js

```diff
@@ -0,0 +1,9 @@
+export default function poolPrioritySort(a, b) {
+  if (a.dsn.priority > b.dsn.priority) {
```

Comment:
You can simplify the entire function to:
`return a.dsn.priority - b.dsn.priority;`
It is the >0, <0, and ==0 that matters for sorting.

Reply:
neat, will update

Reply:
What if the dsn doesn’t have a priority ? Then you have undefined - undefined which is NaN. With explicit less and or greater than checks it would have been 0